cluster/images/hyperkube/master-multi.json

```diff
@@ -21,12 +29,19 @@
       "command": [
               "/hyperkube",
               "apiserver",
+              "--admission-control=NamespaceLifecycle,SecurityContextDeny,LimitRanger,ServiceAccount,ResourceQuota",
```

Comment:
Do you really want `SecurityContextDeny`?  We're moving in the direction of phasing this out (in terms of appearing in the list of defaults)